### PR TITLE
Note/change verify phone number

### PR DESCRIPTION
**Pull request by Panoj-k**
- Connect verification code and new phone number to backend
- Set up alert message after submitting the verification code

image

### CODE REVIEW

File: src/components/organisms/Alert.tsx [hunk 1]
```diff
@@ -44,7 +45,9 @@ const Alert = (props: AlertProps) => {
       <AlertDialog.Content>
         <AlertDialog.CloseButton />
         <AlertDialog.Header>{getMessage()?.header}</AlertDialog.Header>
-        <AlertDialog.Body>{getMessage()?.body}</AlertDialog.Body>
+        <AlertDialog.Body>
+          {getMessage()?.body} {customString}
+        </AlertDialog.Body>
```
Comment (pointmekin) [inline src/components/organisms/Alert.tsx]: try ```{`${getMessage()?.body} ${customString}`}```

File: src/screens/ChangePhoneNumberVerificationScreen.tsx [hunk 2]
```diff
@@ -40,20 +44,49 @@ const ChangePhoneNumberVerificationScreen = ({
...
+    console.log(payload);
+    client
+      .put('/setting/changePhoneNumber', payload)
+      .then(() => {
+        setShowSuccessAlert(true);
+      })
+      .catch((err) => {
+        console.log({ err });
+        setShowErrorAlert(true);
+      });
```
Comment (pointmekin) [inline src/screens/ChangePhoneNumberVerificationScreen.tsx]:
```ts
try {
  const { data } = await client.put('/setting/changePhoneNumber', payload)
  if (data) setShowSuccessAlert(true);
} catch () {
  setShowErrorAlert(true);
}
```

**Review by Panoj-k — COMMENTED** [review]
The line spacing looks good now.
image

File: src/components/organisms/Alert.tsx [hunk 3]
```diff
@@ -44,7 +45,13 @@ const Alert = (props: AlertProps) => {
...
         <AlertDialog.CloseButton />
         <AlertDialog.Header>{getMessage()?.header}</AlertDialog.Header>
-        <AlertDialog.Body>{getMessage()?.body}</AlertDialog.Body>
+        <AlertDialog.Body>
+<<<<<<< HEAD
+          {`${getMessage()?.body} ${customString}`}
+=======
+          {getMessage()?.body} {customString}
+>>>>>>> 5e866f634156063d67454b49f320a111dd17aa38
+        </AlertDialog.Body>
```
Comment (pointmekin) [inline src/components/organisms/Alert.tsx]: Please remove conflict marker

**Review by pointmekin — CHANGES_REQUESTED** [review]

**Review by pointmekin — APPROVED** [review]

**Pull request merged**